app/main.py

```diff
@@ -1,25 +1,95 @@
+from __future__ import annotations
 from abc import ABC
 
 
 class IntegerRange:
-    pass
+    def __init__(
+            self,
+            min_amount: int,
+            max_amount: int
+    ) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(
+            self,
+            owner: SlideLimitationValidator,
+            name: str
+    ) -> None:
+        self.protected_attr = "_" + name
+
+    def __get__(
+            self,
+            instance: SlideLimitationValidator,
+            owner: SlideLimitationValidator
+    ) -> int:
+        return getattr(instance, self.protected_attr)
+
+    def __set__(
+            self,
+            instance: SlideLimitationValidator,
+            value: int
+    ) -> None:
+        if value in range(self.min_amount, self.max_amount + 1):
+            setattr(instance, self.protected_attr, value)
+        else:
+            raise ValueError(f"Value {value} is not in range "
```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one, the closing bracket is also written on a new line

Reply:
Do you mean it in raise here or everywhere in the code?